doc/api/inspector.md

```diff
@@ -0,0 +1,102 @@
+# Inspector
+
+> Stability: 1 - Experimental
+
+The `inspector` module provides an API for interacting with the V8 inspector.
+
+It can be accessed using:
+
+```js
+const inspector = require('inspector');
+```
+
+## Class: inspector.Session
+
+The `inspector.Session` class is a primary means of dispatching messages to
+the V8 inspector back-end and receiving message responses and notifications.
+
+### Constructor: new inspector.Session()
+<!-- YAML
+added: REPLACEME
+-->
+
+Create a new instance of the `inspector.Session` class. The resulted inspector
+session needs to be connected through [`session.connect()`][] before
+messages can be sent to the inspector backend.
+
+`inspector.Session` is is an [`EventEmitter`][] with the following events:
+
+### Event: 'inspectorNotification'
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object} The notification message object
+
+Emitted when a notification from the V8 Inspector is received.
+
+### Event: INSPECTOR_PROTOCOL_METHOD
```

Comment:
This is an interesting doc case, because there are no other examples of it currently in Node that I'm aware of. It is common in node packages to have events with values that aren't known before hand (websockets, etc). I think its not clear enough that its a variable, maybe `<inspector-protocol-method>` would be better, because its clearly not a string?  Also, I  think this would be more clear if  the 'inspectorNotification' event had an example and some more text. IIRC, each inspectorNotification has a 'method: <inspector-protocol-method>' field, and the notification is emitted twice, once with the event name set to the protocol-method. This allows subscribing to all notifications using 'inspectorNotificaion', but it also allows users of the API to subscribe to only the notifications they know they are interested in. <--- something like this last sentence is what is missing. With some more info in the 'inspectorNotification' event, its possible this "pseudo event" would not even need to show up as its own `## Event: ... (something)...` section, or at least could be documented with a simple "See the 'inspectorNotification' event for more information.